### PR TITLE
Add breadcrumbs to subtests

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**

### CODE REVIEW

File: src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import { Spacing } from '../../../styles';
+
+export default function SubtestsBreadcrumbs() {
+  const breadcrumbs = [
+    <div key='1'></div>,
+    <Link key='2' href='/'>
+      Home
+    </Link>,
+    <Link key='3' href='/'>
```
Comment (beatrice-acasandrei) [inline src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx:14]: @julienw Here I was thinking of having a function which generates the previous URL. Do you have other suggestions for me?

File: src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  ];
+
+  return (
+    <Stack spacing={2} direction='row' alignItems='center'>
+      <Breadcrumbs
+        separator={<ChevronLeftIcon fontSize='small' />}
+        aria-label='subtests breadcrumb'
+        sx={{
+          marginLeft: Spacing.xLarge,
+          marginBottom: `${Spacing.layoutXLarge + 4}px`,
```
Comment (julienw) [inline src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx:73]: When using `sx`, it's better to use plain numbers, they are then multiplied by "8" automatically.
But if you add the breadcrumbs in the other component like suggestd above, you'll probably just need the margin bottom (maybe with 2 or 3 as the value)

File: src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      : getPreviousCompareOverTimeURL();
+  const breadcrumbs = [
+    <div key='1'></div>,
+    <Link key='2' href='/'>
+      Home
+    </Link>,
+    <Link key='3' href={allResultsURL}>
+      All results
+    </Link>,
+  ];
```
Comment (julienw) [inline src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx:64]: nit: Don't use a const, add them directly below.
(and then you don't need the key)

File: src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+  const allResultsURL =
+    view === subtestsView
+      ? getPreviousCompareWithBaseURL()
+      : getPreviousCompareOverTimeURL();
+  const breadcrumbs = [
+    <div key='1'></div>,
+    <Link key='2' href='/'>
+      Home
+    </Link>,
+    <Link key='3' href={allResultsURL}>
```
Comment (julienw) [inline src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx:61]: it's not clear to me if we should use a URL or just a "window.close()", but let's keep it this way for now and we can adjust later.

**Review by julienw — COMMENTED** [review]
Hey beatrice, sorry for the delay. Here are some comments for this PR, I think that should be good to go after that!

File: src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  const currentSearchParams = new URLSearchParams(location.search);
+  const interval = currentSearchParams.get('interval');
+  const baseRepo = currentSearchParams.get('baseRepo');
+  const newRev = currentSearchParams.get('newRev');
+  const newRepo = currentSearchParams.get('newRepo');
+  const framework = currentSearchParams.get('framework');
+
+  const params = new URLSearchParams({
+    baseRepo,
+    selectedTimeRange: interval,
```
[the remark this answers is not in the document]
Comment (beatrice-acasandrei) [reply src/components/CompareResults/SubtestsResults/SubtestsBreadcrumbs.tsx:42]: https://mozilla-hub.atlassian.net/browse/PCF-490

**Review by julienw — APPROVED** [review]
Looks good to me, thanks!
Just some nits but you can land once they're resolved

**Pull request merged**